main.py

```diff
@@ -171,33 +256,44 @@
                         element.members.append(buddy)
 
                 viewers = []
+                frozones = []
```

Comment:
where is my supersuit

Reply:
LOL yes, that is exactly what came into my mind too.